writers/mongodb/mongodb.go

```diff
@@ -0,0 +1,61 @@
+package mongodb
+
+import (
+	"context"
+
+	"github.com/mongodb/mongo-go-driver/bson"
+	"github.com/mongodb/mongo-go-driver/mongo"
+
+	"github.com/mainflux/mainflux"
+	"github.com/mainflux/mainflux/writers"
+)
+
+const (
+	dbName         string = "mainflux"
+	collectionName string = "mainflux"
+)
+
+func Connect(addr string, dbName string) (*mongo.Database, error) {
+	client, err := mongo.Connect(context.Background(), addr, nil)
+
+	if err != nil {
+		return nil, err
+	}
+
+	db := client.Database(dbName)
+	return db, nil
+}
+
+var _ writers.MessageRepository = (*mongoRepo)(nil)
+
+type mongoRepo struct {
```

Comment:
Rename to `mongodbRepo`. In order to be consistent, I'll do the same refactoring for InfluxDB.

Reply:
@dusanb94 as I remember Dejan's suggestion, it should go the other way :).